velox/docs/functions/spark/binary.rst

```diff
@@ -15,6 +15,12 @@ Binary Functions
     Computes SHA-1 digest of x and convert the result to a hex string.
     Note: x can only be varbinary type.
 
+.. spark:function:: sha2(x, bitLength) -> varchar
+
+    Calculate SHA-2 family of functions (SHA-224, SHA-256, SHA-384, and SHA-512) and convert the result to a hex string.
+    If asking for an unsupported SHA function, the return value is NULL.
```

Comment:
> If asking for an unsupported SHA function

What does that mean? Are you referring to unsupported values of bitLength argument? Let's explain more clearly and state which values are supported. Let's make sure to add a test case for unsupported values.